extensions-jvm/guava-eventbus/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -31,3 +31,4 @@ metadata:
   - "integration"
   status:
   - "preview"
+  - "deprecated"
```

Comment:
Ok, it's not directly linked to this pr. It's camel-main current updates.

Reply:
removed when based on main